lizmap/modules/presentation/install/install.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @author    3liz
+ * @copyright 2017 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license    Mozilla Public License
+ */
+class presentationModuleInstaller extends jInstallerModule
```

Comment:
you should use the new classes of Jelix 1.8 ` \Jelix\Installer\Module\Installer`